app.js

```diff
@@ -36,6 +46,18 @@ app.get('/', function(req, res, next) {
         res.status(200).json(data);
 	});
 });
+
+app.get('/login', function(req, res,next) {
+  console.log(req.body);
+  res.render('loginform', { title: 'Login to Petcare' });
+});
+
+
+
+app.post('/login', passport.authenticate('local', {successRedirect :'/signup',
```

Comment:
successRedirect should be to /home

Reply:
We dont have a home page yet thats why it temporarily redirected here

Reply:
Added this already